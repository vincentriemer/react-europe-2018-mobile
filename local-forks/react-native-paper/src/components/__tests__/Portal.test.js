/* @flow */

import * as React from 'react';
import renderer from 'react-test-renderer';
import { Text } from 'react-native';
import PortalHost from '../Portal/PortalHost';
import Portal from '../Portal/Portal';

it('renders portal with siblings', () => {
  const tree = renderer
    .create(
      <PortalHost>
        <Text>Outside content</Text>
        <Portal>
          <Text>Portal content</Text>
        </Portal>
      </PortalHost>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
