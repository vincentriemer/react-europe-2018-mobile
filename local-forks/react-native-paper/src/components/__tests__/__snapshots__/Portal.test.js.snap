// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders portal with siblings 1`] = `
<View
  pointerEvents="box-none"
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Outside content
  </Text>
  <View
    pointerEvents="box-none"
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      Portal content
    </Text>
  </View>
</View>
`;
